libswift/Sources/Optimizer/InstructionPasses/SimplifyBeginCOWMutation.swift

```diff
@@ -64,8 +65,10 @@ private func optimizeEmptySingleton(_ beginCOW: BeginCOWMutationInst,
     /// in an infinite loop in SILCombine.
     return
   }
-  let builder = Builder(at: beginCOW, location: beginCOW.location, context)
-  let zero = builder.createIntegerLiteral(0, type: beginCOW.uniquenessResult.type);
+  var builder = Builder(at: beginCOW)
+  let zero = builder.createIntegerLiteral(at: beginCOW.location,
```

Comment:
The location should be passed to the constructor not to the create-functions. We could change this the C++ SILBuilder, too, but it would be a lot of refactoring work.